tools/docker/build_image.sh

```diff
@@ -23,6 +23,7 @@ USAGE:
         ubuntu20.04         Use Ubuntu 20.04 as the base image
         centos8.2           Use CentOS 8.2 as the base image
         aliyunlinux3        Use AliyunLinux 3 as the base image
+	anolis8.4	    USE Anolis 8.4 as the base image
```

Comment:
Please replace "USE" with "Use".

Reply:
OK, it has been replaced in the new commit.